admin/src/tables/FaqsTable.jsx

```diff
@@ -47,10 +47,9 @@ const header = {
 	question: __( 'Question' ),
 	language: __( 'Language' ),
 	status: __( 'Status' ),
-	urls_count: __( 'Assigned URLs' ),
```

Comment:
@mvstepanek count not needed?

Reply:
@jremen we can keep that column there.

Reply:
I removed it as it was two time, as you can see 'Assigned URLs' is already there